xrpl/models/utils.py

```diff
@@ -59,7 +70,11 @@ def new_init(self: _Self, *args: List[Any], **kwargs: Dict[str, Any]) -> None:
             )
         original_init(self, **kwargs)
 
-    # noinspection PyTypeHints
-    cls.__init__ = new_init  # type: ignore
-
+    # For Python v3.10 and above, the KW_ONLY attribute in data_class
```

Comment:
@mvadari I came up with this conditional statement to accomodate different versions of python. Does this look good? Do you have any other ideas in mind?

Reply:
For consistency (and in case they backport the changes), I'd recommend doing the same check you do in `KW_ONLY_DATACLASS`